File: task-03/Go/SubTask2.go
package main

import (
    "io/ioutil"
    "log"
)

func main() {
    data, err := ioutil.ReadFile("input.txt")
    if err != nil {
        log.Fatal(err)
    }

    err = ioutil.WriteFile("output.txt", data, 0644)
    if err != nil {
        log.Fatal(err)
    }
}

File: task-03/C++/SubTask2.cpp
#include <iostream>
#include <fstream>
#include <string>

int main() {
    std::ifstream in_file("input.txt");
    std::ofstream out_file("output.txt");

    std::string data((std::istreambuf_iterator<char>(in_file)), std::istreambuf_iterator<char>());
    out_file << data;

    return 0;
}